src/components/Home.jsx

```diff
@@ -2,8 +2,8 @@ import { NavLink } from 'react-router-dom';
 
 function Header() {
   return (
-    <div className="Header">
-      <h1>Bookstore</h1>
+    <div className="header">
+      <h1>Bookstore CMS</h1>
       <ul>
         <li>
           <NavLink to="/" className="links">
```

Comment:
- [ ] In the design template used, there is a user icon aligned to the right of the header. Yours is missing this icon, kindly implement this because it is a requirement that your design should be identical to that of the Zeplin design. 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/208257400-90686969-d540-4383-ad75-383b894a8294.png)

### Zeplin Design

![image](https://user-images.githubusercontent.com/77106962/208257428-c5a508cc-121e-41bb-8497-690745b5d670.png)

- [ ] The header has a border. Kindly implement this also. 👍